projects/ViTDet/configs/lsj-100e_coco-instance.py

```diff
@@ -0,0 +1,137 @@
+_base_ = [
+    '../../../configs/_base_/default_runtime.py',
+]
+
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = 'data/coco/'
+image_size = (1024, 1024)
+
+file_client_args = dict(backend='disk')
+# comment out the code below to use different file client
+# file_client_args = dict(
+#     backend='petrel',
+#     path_mapping=dict({
+#         './data/': 's3://openmmlab/datasets/detection/',
+#         'data/': 's3://openmmlab/datasets/detection/'
+#     }))
+
+train_pipeline = [
+    dict(type='LoadImageFromFile', file_client_args=file_client_args),
+    dict(type='LoadAnnotations', with_bbox=True, with_mask=True),
+    dict(type='RandomFlip', prob=0.5),
```

Comment:
May add comments

Reply:
it's new config, the new vitdet hasn't been trained yet, I'll use the old version first